pac4j-oidc/src/main/java/org/pac4j/oidc/config/OidcConfiguration.java

```diff
@@ -264,6 +264,9 @@ public void setResponseMode(final String responseMode) {
     }
 
     public String getLogoutUrl() {
+        if(logoutUrl == null && getProviderMetadata().getEndSessionEndpointURI() != null) {
+            return getProviderMetadata().getEndSessionEndpointURI().toString();
```

Comment:
I didn't want to edit the rest of the code without an OK from you @leleuj, but right now I'm doing a toString on an java.net.URI here, only to parse it as a the same class at a later time when it is actually used. Seems inefficient.